pytorch_lightning/callbacks/progress/rich_progress.py

```diff
@@ -289,14 +285,18 @@ def _init_progress(self, trainer):
             self.progress = CustomProgress(
                 *self.configure_columns(trainer),
                 self._metric_component,
-                refresh_per_second=self.refresh_rate_per_second,
+                auto_refresh=False,
```

Comment:
I think this is the fix I proposed in #9647 to @SeanNaren to prevent threading issues in the render function.
Did you check that this might enable #9647 to pass the CI now?

Reply:
Talked to Sean regarding it, it didn't work!

Reply:
Hi @kaushikb11, I'm working on #13937 and debugging it has led me to this PR. Do you remember if there is any particular reason why we changed `auto_refresh` from `True` to `False`? Was it for #10362?